tests/integration/targets/setup_vault_configure/vars/main.yml

```diff
@@ -18,6 +19,15 @@ vault_kv2_multi_api_path: '{{ vault_kv2_mount_point }}/data/{{ vault_kv2_multi_p
 vault_kv2_versioned_api_path: '{{ vault_kv2_mount_point }}/data/{{ vault_kv2_versioned_path }}'
 vault_kv2_delete_api_path: '{{ vault_kv2_mount_point }}/delete/{{ vault_kv2_versioned_path }}'
 vault_kv2_metadata_api_path: '{{ vault_kv2_mount_point }}/metadata/{{ vault_kv2_versioned_path }}'
+vault_kv2_api_list_mount_point: '{{ vault_kv2_mount_point }}/metadata'
+vault_kv2_api_list_path: '{{ vault_kv2_mount_point }}/metadata/{{ vault_kv2_path }}'
+
+vault_policy_api_list_path: 'sys/policies/acl'
+
+vault_kv2_api_list_inexistent_path: '{{ vault_kv2_mount_point }}/metadata/__inexistent'
+vault_kv2_api_list_inexistent_mount_point: '{{ vault_kv2_mount_point }}__inexistent/metadata'
+vault_kv2_api_list_inexistent_unauthorized_path: '{{ vault_kv2_mount_point }}/metadate/__inexistent_no_auth'
```

Comment:
everything else looks good but I'm confused by this change in https://github.com/ansible-collections/community.hashi_vault/pull/343/commits/dd7c6779f1d41ad7b4d214d843ced0adbe04ee8a

Reply:
Yeah me too, not sure what that's about..

Reply:
Have updated in https://github.com/ansible-collections/community.hashi_vault/pull/343/commits/2b43f4c1b7c7972a9ee15a47c4947a26df590418